addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -219,6 +219,7 @@ export class OdooEditor extends EventTarget {
                 toSanitize: true,
                 isRootEditable: true,
                 placeholder: false,
+                pointerInsideCell: true,
```

Comment:
@visp-odoo Do we need actually need a flag? and does it always need to be true ?

Reply:
Yes, we need flag because if i write directly in `_onScroll` then it does't show any effect because if you scroll little `tableUiMenu` will be `visible` again.`onMouse` move we have code for visibling `tableUiMenu` so i set the flag `true` because during the scroll if mouse is out of cell then `tableUiMenu` get `hide` it will not `visible` until you move your mouse.

does it always need to be true ?
=> No, not necessary.